tests/yam/agama/playwright_test.pm

```diff
@@ -0,0 +1,32 @@
+## Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Run interactive installation with Agama,
+# run playwright tests directly from the Live ISO.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+
+use testapi qw(
+  assert_screen
+  assert_script_run
+  get_var
+  select_console
+  upload_logs
+);
+
+sub run {
+    my $testcase = 'take_screenshots';
```

Comment:
it is not a test case, it is called 'test' in playwright, but just put the string on the command, we don't know how we are going to do this in the future, better not declare variables.

Reply:
updated